src/views/studio/studio-info-box.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {connect} from 'react-redux';
+import Button from '../../components/forms/button.jsx';
+
+const StudioInfoBox = ({showInfoBox, onClose, ...props}) => {
+    if (!showInfoBox) return null;
+
+    return (
+        <div className="studio-invitation studio-info-box"> {/* TODO move more styling into studio-info-box? */}
+            {props.children}
+            <Button
+                className="studio-info-close-button"
+                isCloseType
+                onClick={onClose}
+            />
+        </div>
+    );
+};
+
+StudioInfoBox.propTypes = {
+    showInfoBox: PropTypes.bool,
+    onClose: PropTypes.func,
+    children: PropTypes.node
+};
+
+const mapStateToProps = () => ({});
+
+const mapDispatchToProps = () => ({});
+
+export default connect(mapStateToProps, mapDispatchToProps)(StudioInfoBox);
```

Comment:
It looks like we aren't using anything from redux here? Can this be removed

Reply:
Ah yep! I think I forgot to remove it when I stopped using either of those functions.